### DISCUSSION TITLE
Peut on espérer que cette traduction soit un jour intégrée pleinement au jeu ?

### DISCUSSION THREAD

**Discussion by Fizban83** [L-Sherry/French-CC#15] [General]
Tout d'abord, merci

Merci pour cette traduction incroyable. Vous avez fait un boulot de ouf. Donc grand merci.

Et sinon bah juste oui, avez vous pu entrer en contact avec les devs du jeu pour implémenter cette trad fr dans le jeu de base ?

**Comment by L-Sherry** [comment]
En résumé, non.

Déjà, les russes et russophones, qui sont les premiers à avoir traduit entièrement le jeu, ont déjà essayé et globalement, l'éditeur était frileux (ça se comprend, ça donne une relation commerciale déséquilibrée entre eux et les traducteurs, qui pourraient être contestée au tribunal) et les développeurs n'avaient pas envie d'avoir à gérer une traduction supplémentaire.

Ensuite, même si je leur file ma traduction gratuitement et qu'ils l'intègrent au jeu, ils devront payer des taxes à ces vautours de la Spedidam, l'équivalent de la Sacem pour les livres.

Enfin, les développeurs ne travaillent plus sur CrossCode. Il n'y a plus de mises à jour et leur vielle promesse de fournir des outils de traductions s'est envolée. Ils sont à fond sur leur nouveau jeu.

Bref, à mon avis, c'est mort.
